config/initializers/discovery_setup.rb

```diff
@@ -1 +1,3 @@
-
+class Subnet < ActiveRecord::Base
+  attr_accessible :discovery_id
+end
```

Comment:
Oh cool thanks, instead of reopening the class, would you mind implementing the same approach as in https://github.com/theforeman/foreman_discovery/blob/develop/app/models/hostgroup_extensions.rb

(Include it in the `engine.rb`) Thanks!